.github/repo_meta.yaml

```diff
@@ -1,5 +1,5 @@
 # point_of_contact: the owner of this repository, can be a GitHub user or GitHub team
-point_of_contact: @sfc-gh-kmcgrady
+point_of_contact: "@sfc-gh-kmcgrady"
```

Comment:
We might want to change this back to be safe  - matching these examples ([here](https://github.com/snowflakedb/snowflake-sqlalchemy/blob/main/.github/repo_meta.yaml) & [here](https://github.com/snowflakedb/snowflake-connector-net/blob/master/.github/repo_meta.yaml))

Reply:
I think we should be fine here -- quotes are optional in yaml for most strings, but `@` is actually a reserved character that you're technically not allowed to start strings with (see examples 5.9 and 5.10 in the [super-hard-to-read yaml spec](http://yaml.org/spec/1.2-old/spec.html#id2772075).